doc/rados/configuration/ceph-conf.rst

```diff
@@ -493,18 +493,12 @@ present in the monitors' configuration database, use ``ceph config dump``.
 Help
 ====
 
-You can get help for a particular option with:
+To get help for a particular option , run the following command:
```

Comment:
space before comma

Reply:
Oh good Lord. Things like this just might strengthen your rhetorical case against vim.

Reply:
bwah hah hah.  

>I walked in there and I said "Droids, I want to lose.  I mean, I want
to lose.  I want to see line editors on CRTs and nulls in my files.
Write 36 bit ascii that can't be read except with the monitor
filtering it.  I mean LOSE, LOSE, LOSE!"  And I started jumping up and
down yelling "LOSE, LOSE", and Mark Crispin came in and started
jumping up and down with me yelling "Hee, dakedo UNIX nanka wo
tsukatte, umaku ikanakute mo shiranai yo", and a DEC sales rep came
over, put an arm around my shoulder, and said "How'd you like me to
show you a *real* editor that has macros and things like that?  We
have one, it's called TV...."